GoogleUtilities.podspec

```diff
@@ -77,6 +77,15 @@ other Google CocoaPods. They're not intended for direct public usage.
     adss.dependency 'GoogleUtilities/Environment'
   end
 
+  s.subspec 'SceneDelegateSwizzler' do |sdss|
```

Comment:
Cocoapods subspecs are actually broken and we are moving away from using them (will be done at I/O), so we should not introduce new subspecs. I think we should move the files to `AppDelegateSwizzler`.

cc @paulb777

Reply:
+1 Please do not add another subspec

Reply:
Done.